pages/e11y/markdown.lib.js

```diff
@@ -0,0 +1,21 @@
+const {isDev} = require('../views/_data/env');
+const highlight = require('./prismjs.lib');
+
+const MarkdownIt = require('markdown-it');
+const replaceLink = require('markdown-it-replace-link');
+
+const markdown = MarkdownIt({
+  html: true,
+  highlight: highlight,
+  replaceLink: function (link) {
+    const domain = isDev ? 'http://localhost:3005/ui-playground' : 'https://exadel-inc.github.io/ui-playground';
```

Comment:
TODO: (not necessary for current pr) get rid of the hardcoded domain we should be able to get it from the settings